blockchain/standalone/pow_test.go

```diff
@@ -355,9 +356,9 @@ func TestCheckProofOfWork(t *testing.T) {
 			t.Errorf("%q: unexpected err -- got %v, want nil", test.name, err)
 			continue
 		} else if test.err != nil {
-			if !IsErrorCode(err, test.err.(RuleError).ErrorCode) {
+			if !errors.Is(err, test.err) {
```

Comment:
Same here.

Reply:
Hmm, that'd exclude the `hash` test field.

Reply:
I don't think so.  That part would remain.  Only this:

```Go
		err = CheckProofOfWork(hash, test.bits, powLimit)
		if !errors.Is(err, test.err) {
			t.Errorf("%q: unexpected err -- got %v, want %v", test.name, err,
				test.err)
			continue
		}
```